Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'capistrano-s3'
+gem 'dotenv-rails'
```

Comment:
I could be missing something, but do you:

* Do you include setup info for npm in your Readme?
* What about for Ruby?
* Should you specify a Ruby version in this Gemfile?
* Are there deploy instructions?